pyproject.toml

```diff
@@ -1,6 +1,25 @@
+[tool.poetry]
+name = "mppsolar"
+version = "0.9.13"
```

Comment:
I dont know poetry - any way to have the version 'dynamic', so the version only needs to be updated in one place

Reply:
Dear @jblance 
To my knowledge there is no solution for this crucial issue, yet.
There are several discussions going on and solutions are proposed, but I have not seen a sound solution to this issue.
Cheers,
Volker